script/Arcueid/Day 7/07_00_ARC07_3_1_4.txt

```diff
@@ -323,7 +323,7 @@ I don't like to waste energy after all."
 [sha:bd0f72ee128476c7557cd5c06b8f6f58e38704b8]{
 -- Page 645, Offset 18203.
 -- ……名残惜しいが、これ以上アルクェイドを止める理由が思いつかない。
-...I'm reluctant to see it go, but I can't think of any more reasons to stop her.
+...I'm reluctant to see it go, but I can't think of any reasons to stop her.
```

Comment:
There's an ijou in here, I think implication is Shiki's already been stalling her for a while.